csv_consolidator.py

```diff
@@ -0,0 +1,61 @@
+import sys
+import re
+
+def consolidateCSV(csvPath, outputFile):
+    """ Takes the csv file located at csvPath that was downloaded from 
+    https://api.bitcoincharts.com/v1/csv/ and consolidates the duplicate 
+    timestamps into one entry with the price and volume values being the 
+    average of all entries with that timestamp. It is required that the 
+    csv file located at csvPath has the form: 
+        [timestamp], [price], [volume]
+    """
+    print("beginning consolidation of " + csvPath + " to output file " + \
+          outputFile + "...\nthis may take a while...\n")
+
+    with open(csvPath) as input:
+        firstLine = input.readline().split(",")
```

Comment:
local variables usually use underscores and lowercase.
firstLine -> first_line
currentTimeStamp -> current_time_stamp